Makefile

```diff
@@ -153,7 +153,7 @@ cover: nocover
 	@echo pre-compiling tests
 	@time go test -i $(shell go list ./...)
 	# TODO Switch to single `go test` that already supports multiple packages, but watch out for .nocover dirs.
```

Comment:
please remove the above 3 lines. Pre-compilation no longer makes sense since we invoke a single `go test` command anyway, and TODO is done by this PR.

Reply:
I think we also no longer need nocover

Reply:
we do need it, it's protection against adding packages that don't have test files, which are not included in coverage by default.